examples/snap-plugin-collector-rand/rand/rand.go

```diff
@@ -127,7 +155,12 @@ func (RandCollector) GetMetricTypes(cfg plugin.Config) ([]plugin.Metric, error)
 		}
 		metrics = append(metrics, metric)
 	}
-
+	if req {
```

Comment:
No, because `req` is for testing the case where a config is required in order for the plugin to load. `cfg["depsReq"]` is for testing the case when a plugin has dependencies that are not met.